algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableEvidenceFactory.java

```diff
@@ -154,6 +154,8 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
                             .approvalStatus(approvalStatus)
                             .direction(direction)
                             .evidenceUrls(evidenceUrls)
+                            .date(entry.updateDate())
```

Comment:
Would be nice to keep the same order here as in the datamodel? source, date etc.
Same in some other places you altered (don't know if this is really a coding standard, but I think it's nice :) )